### PR TITLE
kickoff meetings outline

### PR DESCRIPTION
**Pull request by Bamieh**
Based on last meeting's discussion.

### CODE REVIEW

File: kickoff_meetings.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- Mentees should drive activity towards career goals (have a “growth attitude”).
+- Mentors should help their mentees refine their goals during the first week of the program.
+- The mentor should mention their expectation reaching the end of the program.
+
+## Availability
+- Meeting dates and times
+- Medium of communication
+
+## Privacy & CoC
+- determine what should be confidential and what can be disclosed.
```
Comment (detrohutt) [inline kickoff_meetings.md]: - Determine upfront what should be confidential and what can be disclosed, or designate a process such as having private sessions at end of meetings when needed.

File: kickoff_meetings.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- Mentee and mentor are comfortable recording sessions
```
Comment (detrohutt) [inline kickoff_meetings.md]: - Determine if mentee and mentor are comfortable recording sessions

**Review by detrohutt — APPROVED** [review]
My changes are just suggestions. Feel free to ignore. Great work! LGTM!

File: kickoff_meetings.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- Mentor and mentee should discuss the mentee's goals and expectations
+- Mentees should drive activity towards career goals (have a “growth attitude”).
+- Mentors should help their mentees refine their goals during the first week of the program.
+- The mentor should mention their expectation reaching the end of the program.
+
+## Availability
+- Meeting dates and times
+- Medium of communication
+
+## Privacy & CoC
```
Comment (dshaw) [inline kickoff_meetings.md:29]: Add a line to review and confirm that all meetings and interactions will be done with respect to [Node.js Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md).

File: kickoff_meetings.md (hunk 1 above)
Comment (dshaw) [reply kickoff_meetings.md]: Nit: Capitalize first letter of this line.

**Review by dshaw — CHANGES_REQUESTED** [review]
I think we should include specific, proactive engagement with the Node.js Code of Conduct.

File: kickoff_meetings.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- Review and confirm that all meetings and interactions will be done with respect to [Node.js Code of Conduct][]
```
Comment (detrohutt) [inline kickoff_meetings.md:30]: `- Review and confirm that all meetings and interactions will adhere to the [Node.js Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md)`

File: kickoff_meetings.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+- Determine upfront what should be confidential and what can be disclosed, or designate a process such as having private sessions at end of meetings when needed.
+- Determine if the mentee and mentor are comfortable recording sessions
+
```
Comment (detrohutt) [inline kickoff_meetings.md:33]: I think everything below this point should be deleted. If you were wanting an extra link to the CoC besides the one above, the markup needs to be fixed to:

`[Node.js Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md)`

**Review by detrohutt — CHANGES_REQUESTED** [review]

File: kickoff_meetings.md (hunk 4 above)
Comment (Bamieh) [reply kickoff_meetings.md:30]: @detrohutt this syntax is valid, I prefer to have the links at the bottom of the markdown file for faster refactoring/editing.

Comment (detrohutt) [reply kickoff_meetings.md:30]: Interesting!

**Review by dshaw — APPROVED** [review]
LGTM. Thank you.

**Review by detrohutt — APPROVED** [review]

**Pull request merged**